packages/core-ethereum/src/channel.ts

```diff
@@ -77,11 +76,16 @@ class Channel {
     }
   }
 
+  setCommitment(): Promise<void> {
+    return this.commitment.initialize()
```

Comment:
Let's just inline this.commitment.initialize() rather than having this method

Reply:
removed, no longer necessary